### PR TITLE
DPLT-1119 status page subscription

### LINKED ISSUE #222: Logs widget automatic refresh with subscriptions

### PR DESCRIPTION
**Pull request by roshaans**
The indexer status page uses websocket connection to update its state.

### CODE REVIEW

File: frontend/widgets/src/QueryApi.IndexerStatus.jsx [hunk 1]
```diff
@@ -105,8 +105,180 @@ State.init({
...
+    const data = JSON.parse(e.data);
+    console.log("received data", data);
+    if (data.type === "data" && data.id === "logs") {
+      processLogs(data.payload.data);
+    }
+  };
+
+  ws.onerror = (err) => {
+    State.update({ ws_logs: null });
+    console.log("WebSocket error", err);
```
[the remark this answers is not in the document]
Comment (roshaans) [reply frontend/widgets/src/QueryApi.IndexerStatus.jsx:247]: Nope, only console.log exists on the VM as of now

File: frontend/widgets/src/QueryApi.IndexerStatus.jsx [hunk 2]
```diff
@@ -131,7 +305,7 @@ const sanitizedFunctionName = indexer_name;
 const fullFunctionName = accountName + "_" + sanitizedFunctionName;
 const logsDoc = `
   query QueryLogs($offset: Int) {
-    indexer_log_entries(order_by: {timestamp: desc}, limit: ${LIMIT}, offset: $offset, where: {function_name: {_eq: "${accountId}/${indexer_name}"}}) {
+    indexer_log_entries(order_by: {block_height: desc}, limit: ${LIMIT}, offset: $offset, where: {function_name: {_eq: "${accountId}/${indexer_name}"}}) {
```
Comment (gabehamilton) [inline frontend/widgets/src/QueryApi.IndexerStatus.jsx:308]: What's the reason for this change?

Comment (roshaans) [reply frontend/widgets/src/QueryApi.IndexerStatus.jsx:308]: This should actually stay as timestamp rather than block_height. Will take care of this
